packages/composites/dydx-rewards/src/method/formulas/initial.ts

```diff
@@ -0,0 +1,163 @@
+import * as bn from 'bignumber.js'
+import { AddressRewards, OracleRewardsData } from '../../ipfs-data'
+import { addReward } from '../poke'
+import { BigNumber } from 'ethers'
+
+// We expect the amount with 2 decimal points, and the token has 18 decimals.
+const rewardAmountToBigNumber = (amount: number): BigNumber =>
+  BigNumber.from(amount * 100).mul(BigNumber.from(10).pow(16))
+
+// Retroactive tiers are sorted descending to make it easier to find the highest applicable tier
+const retroactiveTiers = [
+  {
+    min: 1_000_000,
+    reward: rewardAmountToBigNumber(9529.86),
+    volumeRequirement: 100_000,
+  },
+  {
+    min: 100_000,
+    reward: rewardAmountToBigNumber(6413.91),
+    volumeRequirement: 50_000,
+  },
+  {
+    min: 10_000,
+    reward: rewardAmountToBigNumber(4349.63),
+    volumeRequirement: 5_000,
+  },
+  {
+    min: 1,
+    reward: rewardAmountToBigNumber(1163.51),
+    volumeRequirement: 500,
+  },
+  {
+    min: 0,
+    reward: rewardAmountToBigNumber(310.75),
```

Comment:
Where did you pull these numbers from? Can you add a link to them in the comments around line 10?

Reply:
These numbers come from internal documentation I cannot link publicly. I'll have a walkthrough of this with you and Justin next week.

Reply:
Understood, yes that would be great